tests/check_doctest_names.py

```diff
@@ -0,0 +1,201 @@
+"""A helper script to check names in doctests.
+
```

Comment:
I'd like one line that includes how to call it right at the top:
```
Intended to called from pyvista's root directory with
python tests/check_doctest_names.py
```

Since we'll use this for testing, we might as well include how to call it.  It's not explicitly necessary.

Reply:
Perfect.